src/software/thunderscope/gl/layers/gl_cost_vis_layer.py

```diff
@@ -94,7 +94,7 @@ def __init__(self, name: str, buffer_size: int = 5) -> None:
         )
         self.cached_world = World()
         self.cached_cost_vis = CostVisualization()
-        self.timeout = time.time() + GLCostVisLayer.COST_VISUALIZATION_TIMEOUT_S
+        self.timeout = 0.0
```

Comment:
why do we need this change?

Reply:
Initially I thought this fixed a bug, but I think I was wrong. I've reverted it.